x-pack/plugins/ml/server/models/data_recognizer/modules/metrics_ui_k8s/ml/k8s_cpu_usage.json

```diff
@@ -0,0 +1,53 @@
+{
+    "job_type": "anomaly_detector",
+    "groups": [
+      "k8s",
+      "metrics"
+    ],
+    "description": "Metrics: Kubernetes - Identify unusual spikes in cpu utilization across kubernetes pods.",
+    "analysis_config": {
+      "bucket_span": "15m",
+      "detectors": [
+        {
+          "detector_description": "max('kubernetes.pod.cpu.usage.node.pct') partitionfield='kubernetes.namespace'",
```

Comment:
Is it worth removing the partitionfield part from the description here? Otherwise, the metrics UI will need to ensure they also override the `detector_description` if they override the `partition_field_name`?

Reply:
yep, removed these in https://github.com/elastic/kibana/pull/76460/commits/5f013319cc34f6862ad17f9e7c27e0f1385da257